aula-css/exercicios/photo-grid/aula-1/index.html

```diff
@@ -30,8 +30,9 @@ <h1>Laís' Photo Grid</h1>
 
             <div class="item">
                 <img src="imagens/IMG_20191029_131349_132.jpg" alt="mulher">
```

Comment:
```suggestion
                <img src="imagens/IMG_20191029_131349_132.jpg" alt="mulher" />
```

Lembrar que tags <img> precisam dessa barrinha no final para fechamento da tag.